website/pages/docs/security/index.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
-page_title: Security Model
-sidebar_title: Security Model
+page_title: Security
```

Comment:
Should this be changing names? In nomad, this is still [security model](https://www.nomadproject.io/docs/internals/security)

Reply:
Yes, it is changing names based on the new restructuring. Some things are deviating from what has been consistent in the past